service/controller/v14/resource/bridgezone/resource.go

```diff
@@ -0,0 +1,415 @@
+package bridgezone
+
+import (
+	"context"
+	"reflect"
+	"strings"
+
+	"github.com/aws/aws-sdk-go/service/route53"
+	"github.com/giantswarm/microerror"
+	"github.com/giantswarm/micrologger"
+	"k8s.io/client-go/kubernetes"
+
+	"github.com/giantswarm/aws-operator/client/aws"
+	"github.com/giantswarm/aws-operator/service/controller/v14/controllercontext"
+	"github.com/giantswarm/aws-operator/service/controller/v14/credential"
+	"github.com/giantswarm/aws-operator/service/controller/v14/key"
+)
+
+const (
+	name = "bridgezonev14"
+)
+
+type Config struct {
+	HostAWSConfig aws.Config
+	HostRoute53   *route53.Route53
+	K8sClient     kubernetes.Interface
+	Logger        micrologger.Logger
+
+	Route53Enabled bool
+}
+
+// Resource is bridgezone resource making sure we have fallback delegation in
+// old DNS structure. This is only for the migration period. When we delete the
+// "intermediate" zone this resource becomes noop and we do not need it
+// anymore.
+//
+// Old structure looks like:
+//
+//	installation.eu-central-1.aws.gigantic.io (host account)
+//	└── NS k8s.installation.eu-central-1.aws.gigantic.io (default guest account)
+//	    ├── A api.old_cluster.k8s.installation.eu-central-1.aws.gigantic.io
+//	    └── A ingress.old_cluster.k8s.installation.eu-central-1.aws.gigantic.io
+//
+// New structure looks like:
+//
+//	installation.eu-central-1.aws.gigantic.io (host account)
+//	└── NS new_cluster.k8s.installation.eu-central-1.aws.gigantic.io (byoc guest account)
+//	    ├── A api.new_cluster.k8s.installation.eu-central-1.aws.gigantic.io
+//	    └── A ingress.new_cluster.k8s.installation.eu-central-1.aws.gigantic.io
+//
+// For the migration period for new clusters we need also to add delegation to
+// k8s.eu-central-1.aws.gigantic.io because of the AWS DNS caching issues.
```

Comment:
Missing `installation` in dns name should be > `k8s.installation.eu-central-1.aws.gigantic.io`